run.cmd

```diff
@@ -17,12 +17,11 @@ if not defined VisualStudioVersion (
 set Platform=
 
 :: Restore the Tools directory
-call %~dp0init-tools.cmd
+call %~dp0init-tools.cmd -quiet
```

Comment:
It's slightly disconcerting to not get the reassuring "Tools are already initialized" or "Done initializing tools" messages, but that hand holding isn't truly necessary, and additional information is still available with a bit more work ("set _echo=on" or checking init-tools.log).  It's also difficult not to add the inverse switch, "verbose", but I understand and agree with your principal of reducing these scripts, not bloating them.